paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/EditCardDetailsInteractor.kt

```diff
@@ -253,24 +268,30 @@ internal class DefaultEditCardDetailsInteractor(
         }
     }
 
-    private fun buildDefaultCardEntry(): CardDetailsEntry {
+    private fun CardEditConfiguration.buildDefaultCardEntry(): CardDetailsEntry {
         return CardDetailsEntry(
             cardBrandChoice = defaultCardBrandChoice(),
             expiryDateState = defaultExpiryDateState(),
         )
     }
 
-    private fun defaultCardBrandChoice() = payload.getPreferredChoice(cardBrandFilter)
+    private fun CardEditConfiguration.defaultCardBrandChoice(): CardBrandChoice {
+        return if (cardEditConfiguration != null) {
+            payload.getPreferredChoice(cardEditConfiguration.cardBrandFilter)
+        } else {
+            CardBrandChoice(Unknown, enabled = false)
+        }
+    }
 
-    private fun defaultExpiryDateState(): ExpiryDateState {
+    private fun CardEditConfiguration.defaultExpiryDateState(): ExpiryDateState {
         return ExpiryDateState.create(
             editPayload = payload,
-            enabled = areExpiryDateAndAddressModificationSupported
+            enabled = areExpiryDateAndAddressModificationSupported == true
```

Comment:
This can be changed because `CardEditConfiguration` is the receiver.
```suggestion
            enabled = areExpiryDateAndAddressModificationSupported
```